cpp/cuio/kafka/tests/kafka_consumer_tests.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <map>
+#include <memory>
+#include <string>
+#include "kafka_consumer.hpp"
+
+#include <cudf/io/datasource.hpp>
+#include <cudf/io/functions.hpp>
+#include <cudf/strings/string_view.cuh>
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/table/table.hpp>
+#include <cudf/table/table_view.hpp>
+
+#define CUDF_DATASOURCE_TEST_PROGRAM_MAIN() \
```

Comment:
Simply because I was including the source or linking against the cudf "tests" so it wasn't there. If you think its worth it I can include them and link against the "cudftestutil"

Reply:
I receive an error for a missing main method if I remove it ... 

```[100%] Linking CXX executable ../../../gtests/CUDF_KAFKA_HOST_READ
/usr/lib/gcc/x86_64-linux-gnu/7/../../../x86_64-linux-gnu/Scrt1.o: In function `_start':
(.text+0x20): undefined reference to `main'
```

Reply:
link to `gtest_main `and you should be good:
https://github.com/google/googletest/blob/master/googletest/docs/primer.md#writing-the-main-function

Reply:
ahh ok